packages/ion-did-provider/__tests__/ion-did-provider.test.ts

```diff
@@ -214,7 +214,7 @@ const createIdentifierOpts = {
     {
       kid: 'did1-test',
       purposes: [IonPublicKeyPurpose.Authentication, IonPublicKeyPurpose.AssertionMethod],
-    },
+      },
```

Comment:
prettier should pick this up

Reply:
Hmzz, had to do with a revert of a commit. Fixed